### PR TITLE
Add new doc: usage guide

### PR DESCRIPTION
**Pull request by microbouji**
As a newcomer myself, I think this getting started guide is something that would make sense as an intro to the different tools and use cases.

Two similar guides can probably be made as overviews to the "Usage" and "Tooling" sections that explain things like babel-node, babel-standalone, babel-runtime etc. for the usage guide, and babylon, code-frame etc. for the tooling guide.

Again, I'm still learning all of this stuff myself, so I'm happy to be more involved with docs that need to be written.

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit aa5bc1971d0e10312b19be32ef007fde27cd05f3

https://deploy-preview-1588--babel-new.netlify.com

**Comment by xtuc** [comment]
Thanks @microbouji! sorry for the delai.

I'll commit a couple of changes in your PR if you don't mind

**Comment by microbouji** [comment]
Sure, feel free to edit liberally. I was trying to balance the tone between a usage guide and a docs page, but may have even slipped to tutorial style at some places.

**Comment by xtuc** [comment]
I've made a couple of minor changes in https://github.com/babel/website/pull/1588/commits/b93db0ef34d315ce85bf61fc128d183f1849269a. Could you please review them if you have some time?

I really like how it's written, thanks!

**Comment by xtuc** [comment]
Sometimes It overlaps with the https://new.babeljs.io/docs/en/next/index.html page, i'll remove some sections of it I think

**Comment by microbouji** [comment]
All changes looking good.

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# Babel Roadmap
+---
+id: roadmap
```
Comment (hzoo) [inline docs/roadmap.md]: Oops I added this in and merged so there's a conflict?

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
+---
+id: usage
+title: Usage Guide
+---
+
+There are quite a few tools in the Babel toolchain that try to make it easy for you to use Babel whether you're an "end-user" or building an integration of Babel itself. You can read more about the tools you decide to use in the "Usage" section of the docs.
+
+If you're building an app or library, there are two main things Babel can do for you:
```
Comment (hzoo) [inline docs/usage.md]: two to -> three or change the wording?

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+- Transform syntax
+- Polyfill features that are missing in your target environment
+- Code transformations (codemods)
+
+This short guide will show you how to compile your JavaScript application code that uses ES2015+ syntax into code that works in the last 2 versions of all browsers, as well as Safari versions 7 and up. That will involve both transforming new syntax and polyfilling missing features.
```
Comment (hzoo) [inline docs/usage.md]: I guess due to the whole "2 last versions" issue maybe we use another example? I'd rather just hardcode the browsers anyway personally (`chrome: 62`), shrug

File: docs/usage.md (hunk 2 above)
Comment (xtuc) [reply docs/usage.md]: No, espically since it's wrong. I jsust changed it for: here are the main things Babel can do for you.

**Comment by hzoo** [comment]
Awesome start, I think we should change the "browsers" part for env, and then maybe think about linking to specific tools for thing like react, ember, vue etc if it already does it for them

**Pull request merged**